hack/parallel.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2020 Red Hat, Inc.
+#
+
+NAMESPACE=${KUBEVIRT_INSTALLED_NAMESPACE:-kubevirt}
+TEMP_FILE=$(mktemp -p /tmp -t kubevirt.deploy.XXXX)
+
+trap 'rm -f $TEMP_FILE' EXIT ERR SIGINT SIGTERM SIGSTOP KILL
```

Comment:
You can't trap SIGKILL

Reply:
i would also remove ERR, or change the shebang to -E, else it wont catch ERR in functions from what i read, and we also have EXIT anyway

Reply:
done